task/02-numbers-tasks.js

```diff
@@ -160,7 +160,7 @@ function parseNumberFromString(value) {
  *   1,2,3   => 3.741657386773941
  */
 function getParallelipidedDiagonal(a,b,c) {
-    throw new Error('Not implemented');
+    return Math.sqrt(Math.pow(a,2)+Math.pow(b,2)+Math.pow(c,2));
 }
```

Comment:
Well done! JFYI: in es6 there is a new method, that does exactly what's required: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot